jekyll/_cci2/executor-types.md

```diff
@@ -34,7 +34,7 @@ Virtual Environment | `docker` | `machine`
  Build Docker images | Yes <sup>(1)</sup> | Yes
  Full control over job environment | No | Yes
  Full root access | No | Yes
- Run multiple databases | No | Yes
+ Run multiple databases | Yes | Yes
```

Comment:
we should add a note about the limitation/resource constraint for performance (maybe add an asterisk here in the table and note it below).

Reply:
can do!